moped-editor/src/views/projects/newProjectView/ProjectSaveButton.js

```diff
@@ -49,27 +19,46 @@ export default function ProjectSaveButton({
   disabled,
   buttonOptions = {},
 }) {
-  const classes = useStyles();
-
-  const buttonClassname = clsx({
-    [classes.buttonSuccess]: success,
-  });
-
   return (
-    <div className={classes.wrapper}>
+    <Box
+      sx={{
+        margin: 1,
+        position: "relative",
+        marginRight: 0,
+      }}
+    >
       <Button
         variant="contained"
         color="primary"
-        className={buttonClassname}
         disabled={(loading && !success) || disabled}
         onClick={success ? null : handleButtonClick}
+        sx={{
+          ...(success && {
+            backgroundColor: green[500],
+            "&:hover": {
+              backgroundColor: green[500],
+            },
+            disabled: true,
+            cursor: "default",
+          }),
+        }}
         {...buttonOptions}
       >
         {success ? <Icon>check</Icon> : label}
       </Button>
       {loading && (
-        <CircularProgress size={24} className={classes.buttonProgress} />
+        <CircularProgress
+          size={24}
+          sx={{
+            color: green[500],
+            position: "absolute",
+            top: "50%",
+            left: "50%",
+            marginTop: -12,
+            marginLeft: -12,
```

Comment:
@mateoclarke I noticed that this fancy green loading spinner has some CSS to make it overlap the button that shows "Create" and then the check icon when the project saves. I think these 12s are being interpreted as theme spacing because the spinner is now out of alignment. could you check this out? 🙏

<img width="193" height="158" alt="Image" src="https://github.com/user-attachments/assets/42ca1a02-2615-4413-a98a-edb45f8b0711" />